src/foam/dao/EasyDAO.js

```diff
@@ -576,6 +624,12 @@ return delegate;
         dao = decorated;
       }
 
+      if ( this.order ) {
```

Comment:
Didn't you already do this above? Also, order isn't an array.

Reply:
@kgrgreer one is java, this is javascript.